sale_order_line_discount_validation/data/mail_template.xml

```diff
@@ -0,0 +1,24 @@
+<odoo noupdate="1">
+
+    <record id="email_template_request_approval" model="mail.template">
+        <field name="name">Sales Order: Request approval</field>
+        <field name="email_from" />
+        <field name="subject" />
+        <field name="model_id" ref="sale.model_sale_order" />
+        <field name="auto_delete" eval="True" />
+        <field name="body_html">
+           <![CDATA[
+                <p>Hello,</p>
+                % if object.state == 'waiting_approval':
+                    Kindly review sale order ${object.name}.
+                % elif object.state == 'approved':
+                    Sale order ${object.name} is approved.
+                % elif object.state == 'draft':
+                    Sale order ${object.name} is refused.
+                % endif
+
```

Comment:
@Vandan-OSI Can you add the link to the sale order in the backend here?

CC: @murtuzasaleh